src/components/Rockets.module.css

```diff
@@ -48,6 +48,18 @@
   box-shadow: 2px 2px 5px  #007bff;
 }
 
+.badges {
+  background: #09c59c;
```

Comment:
[OPTIONAL] Good job styling the badge. For style uniformity, consider using a background of `rgb(0, 153, 174)`. This will enable your design to match that given😉